sys-devel/clang/clang-4.0.1.ebuild

```diff
@@ -164,6 +164,12 @@ multilib_src_configure() {
 		)
 	fi
 
+	if use prefix; then
```

Comment:
Why are you using this awful ugly implicit USE flag instead of checking for non-null `EPREFIX`?

Reply:
Well, that's what most packages are doing. If you prefer a different way, just let me know. This could  be added unconditionally without breaking anything either.

Reply:
I agree that for cases like this, checking for a non-null prefix is nicer.